src/api/validators/biometrics/blood.oxygen.saturation.validator.ts

```diff
@@ -0,0 +1,218 @@
+            .run(request);
+
+        await query('MaxValue').optional()
+            .trim()
+            .run(request);
+
+        await query('RecordedByUserId').optional()
+            .trim()
+            .escape()
+            .isUUID()
+            .run(request);
+
+        await query('createdDateFrom').optional()
+            .trim()
+            .escape()
+            .toDate()
+            .run(request);
+
+        await query('createdDateTo').optional()
+            .trim()
+            .escape()
+            .toDate()
+            .run(request);
+
+        await query('orderBy').optional()
+            .trim()
+            .escape()
+            .run(request);
+
+        await query('order').optional()
+            .trim()
+            .escape()
+            .run(request);
+
+        await query('pageIndex').optional()
+            .trim()
+            .escape()
+            .isInt()
+            .run(request);
+
+        await query('itemsPerPage').optional()
+            .trim()
+            .escape()
+            .isInt()
+            .run(request);
+
+        const result = validationResult(request);
+        if (!result.isEmpty()) {
+            Helper.handleValidationError(result);
+        }
+
+        return BloodOxygenSaturationValidator.getFilter(request);
+    };
+
+    static update = async (request: express.Request): Promise<BloodOxygenSaturationDomainModel> => {
+
+        const id = await BloodOxygenSaturationValidator.getParamId(request);
+        await BloodOxygenSaturationValidator.validateBody(request);
+
+        const domainModel = BloodOxygenSaturationValidator.getDomainModel(request);
+        domainModel.id = id;
+
+        return domainModel;
+    };
+
+    private static async validateBody(request) {
+
+        await body('PatientUserId').optional()
+            .trim()
+            .escape()
+            .isUUID()
+            .run(request);
+
+        await body('BloodOxygenSaturation').optional()
+            .trim()
+            .escape()
+            .toInt()
+            .run(request);
+
+        await body('Unit').optional()
+            .trim()
+            .run(request);
+
+        await body('RecordDate').optional()
+            .trim()
+            .escape()
+            .isDate()
+            .run(request);
+
+        await body('RecordedByUserId').optional()
+            .trim()
+            .escape()
+            .run(request);
+
+        const result = validationResult(request);
+        if (!result.isEmpty()) {
+            Helper.handleValidationError(result);
+        }
+    }
+
+    private static getFilter(request): BloodOxygenSaturationSearchFilters {
+        const pageIndex = request.query.PageIndex !== 'undefined' ? parseInt(request.query.PageIndex as string, 10) : 0;
+
+        const itemsPerPage =
+            request.query.ItemsPerPage !== 'undefined' ? parseInt(request.query.ItemsPerPage as string, 10) : 25;
+
+        const filters: BloodOxygenSaturationSearchFilters = {
```

Comment:
Anything we add to request params and query params such as ids should be camelCase.
i.e. 
request.query.**patientUserId**
 request.query.**minValue**.